src/shell/scripts/omp.zsh

```diff
@@ -48,7 +48,10 @@ function self-insert() {
     zle .self-insert
     return
   fi
-  tooltip=$(::OMP:: print tooltip --config="$POSH_THEME" --shell=zsh --command="$BUFFER" --shell-version="$ZSH_VERSION")
+  # trigger a tip check only if the input is a space character
+  if [[ "$KEYS" = " " ]]; then
```

Comment:
Can't we do this check in `self_insert`? That way we don't reload unnecessarily.

Reply:
Sorry, I don't quite understand but where is `self_insert`? There is only `self-insert` and this check is done in it.

Reply:
I'm sorry, should've looked at this on a bigger display 🤦🏻‍♂️

Reply:
Fine. So is there anything I should change? 😄